spec/rails_helper.rb

```diff
@@ -7,6 +7,7 @@
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
+require 'support/factory_bot'
```

Comment:
We did not require 'support/factory_bot' - we put the code in that file directly into the rails_helper. Either works, just noting it for the conflict eventually, and so we do not have duplicate ways to make this work.